libs/gi/dm/src/dm/character/AvatarPromoteExcelConfigData.ts

```diff
@@ -54,19 +57,38 @@ type AvatarPromoteExcelConfigData = {
 const ascensionSrc = JSON.parse(
   readDMJSON('ExcelBinOutput/AvatarPromoteExcelConfigData.json')
 ) as AvatarPromoteExcelConfigData[]
-export type AscensionData = {
-  [AvatarPromoteId: number]: {
-    props: { [key: string]: number }
-  }[]
-}
+
+export type AscensionRecord = {
+  props: { [key: string]: number }
+  scoinCost: number
+  costItems: Array<
+    | {
+        id: number
+        count: number
+      }
+    | Record<string, never>
```

Comment:
whats the point of this?

Reply:
That an empty object is possible, not sure what would be the better solution